src/logic/otp2/liveStatus/texts.ts

```diff
@@ -0,0 +1,37 @@
+import { Locale, Translation } from '../../../utils/locale'
+
+export const minutesDelayed = (minutes: number): Translation => ({
+    [Locale.BOKMAL]: `${minutes} minutter forsinket`,
+    [Locale.NYNORSK]: `${minutes} minutt forseinka`,
+    [Locale.ENGLISH]: `${minutes} minutes delayed`,
+})
+
+export const onTime: Translation = {
+    [Locale.BOKMAL]: 'I rute',
+    [Locale.NYNORSK]: 'I rute',
+    [Locale.ENGLISH]: 'On time',
+}
+
+export const arrivedStop = (name: string, time: string): Translation => ({
+    [Locale.BOKMAL]: `Stoppet ved ${name} kl. ${time}`,
+    [Locale.NYNORSK]: `Stoppa ved ${name} kl. ${time}`,
+    [Locale.ENGLISH]: `Stopped by ${name} at ${time}`,
```

Comment:
Småpirk, men 'Stopped at' høres mer riktig it i mine ører hvis det er et busstopp.

Reply:
Man bruker vel ofte 'called at', men kanskje mest på britisk engelsk... https://english.stackexchange.com/questions/549245/call-at-vs-stop-at-vs-serve

Reply:
Men definitivt at isf by.